web/src/components/TraceQueryBrowser.css

```diff
@@ -6,6 +6,6 @@
     margin-top: 2rem
 }
 
-.query-browser-input-button{
+.query-browser-input-button {
```

Comment:
Query browser might be a too generic name, since query browser is a term used across the OpenShift Console. We could run into CSS colissions later

Reply:
changed to add the prefix `tracing-*` to names in this file (e.g. `tracing-query-browser-input-button`)